webapp/components/FilterPosts/FilterPosts.spec.js

```diff
@@ -97,13 +107,31 @@ describe('FilterPosts.vue', () => {
       expect(mutations['posts/TOGGLE_CATEGORY']).toHaveBeenCalledWith({}, 'cat4')
     })
 
+    it('calls TOGGLE_LANGUAGE when clicked', () => {
+      const wrapper = openFilterPosts()
+      englishButton = wrapper
+        .findAll('button.language-buttons')
+        .at(languages.findIndex(l => l.code === 'en'))
```

Comment:
I really like your way of doing this! one thing I complain about with vue-test-utils over enzyme for example is that there is not `filterWhere`... but I really like how you've done this... I need to refactor some of my tests that look for the button in a position, which is easy to break if someone adds another button, for example.